influx/influx.go

```diff
@@ -45,10 +45,10 @@ type testInfo struct {
 	testStartTime   time.Time
 }
 
-type userLineData struct {
-	timestamp time.Time
-	scenario  string
-	status    string
+type UserLineData struct {
```

Comment:
You’re absolutely right! 
I decided to export this type so that it can be reused in `userlineparser.go`. 
Additionally, we can simplify the function signature slightly by changing:

```go
func SendUserLineData(timestamp time.Time, scenario, status string)
```

to:

```go
func SendUserLineData(userLineData UserLineData)
```